### ISSUE TITLE
Autostart not working in Appimage.

### ISSUE THREAD

**Issue by abcfy2** [shadowsocks/shadowsocks-qt5#661]
I find if I start shadowsocks-qt5 from appimage, and use autostart, it will create a `~/.config/autostart/shadowsocks-qt5.desktop` with this content:

```ini
[Desktop Entry]
Name=Shadowsocks-Qt5
Exec=/tmp/.mount_shadowlQJoRC/usr/bin/ss-qt5
Type=Application
Terminal=false
X-GNOME-Autostart-enabled=true
```

Because `/tmp` will auto clear after reboot, so this auto start not working.

**Comment by librehat** [comment]
As a workaround, please change the Exec to be your AppImage path.

This is a tricky bug to fix

**Comment by meoow** [comment]
Yeah I think it's more like an AppImage issue, 'cause any apps who can create a autostart shortcut by themselves are affected by this.

It's not really up to ss-qt5 developers to fix this.
